express/blocks/breadcrumb/breadcrumb.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable import/named, import/extensions */
+
+import {
+  createTag,
+} from '../../scripts/scripts.js';
+
+function capitalizeFirstLetter(string) {
+  const words = string.split(' ').map((word) => word[0].toUpperCase() + word.substring(1));
+  return words.join(' ');
+}
+
+export function decorateBreadcrumb($block) {
+  const pathArray = Array.from(window.location.pathname.split('/')).splice(1);
+  let previousCrumb = '';
+  const urlArray = pathArray.map((path) => {
+    previousCrumb = `${previousCrumb}/${path}`;
+    return previousCrumb;
+  });
+  const $homeCrumb = createTag('a', { href: `${urlArray[0]}` });
+  $homeCrumb.textContent = 'Home / ';
```

Comment:
How would this be internationalized? For now we could use the placeholder sheets to store the most common parent crumb titles (fortunately, the hierarchy isn't that deep)

Search for `fetchPlaceholders()` to find example usages.